docs/en/latest/discovery/kubernetes.md

```diff
@@ -318,7 +318,10 @@ metadata:
  name: apisix-test
 rules:
 - apiGroups: [ "" ]
-  resources: [ endpoints,endpointslices ]
+  resources: [ endpoints ]
+  verbs: [ get,list,watch ]
+- apiGroups: [ "discovery.k8s.io" ]
+  resources: [ endpointslices ]
```

Comment:
Based on the documentation, this feature only reached stable status in Kubernetes `v1.21`. 

Did the original documentation work correctly in older versions?